### PR TITLE
(DEV-3882, DEV-3888, DEV-3889) Removes Old Prayer Components

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Removes old components

### How do I test this PR?

It's gonna seem pretty bare...

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js [hunk 1]
```diff
@@ -69,39 +51,24 @@ class PrayerCard extends PureComponent {
...
   render() {
     const {
       interactive,
-      actionsEnabled,
       showHelp,
       header,
-      expanded,
-      avatarSource,
       avatarSize,
       created,
```
Comment (richarddubay) [inline packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js]: If what I'm seeing in UserPrayerList is true, then this should be removed.

File: packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js [hunk 2]
```diff
@@ -113,78 +80,56 @@ class PrayerCard extends PureComponent {
...
-                cancelButtonIndex={buttons.length - 1}
-                destructiveButtonIndex={buttons // ActionSheet only allows for one destructive button // this will only make the first option listed as destructive turn red
-                  .map((option) => option.destructive)
-                  .indexOf(true)}
-                onPress={(index) => handleOnPress(index)}
+    return (
+      <Card>
+        {interactive ? (
+          <HeaderView>
+            <GreyH5>{created ? moment(created).fromNow() : ''}</GreyH5>
```
Comment (richarddubay) [inline packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js]: This should be `prayer.enteredDateTime`, right?

Comment (richarddubay) [reply packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js]:
```suggestion
            <GreyH5>{prayer.enteredDateTime ? moment(prayer.enteredDateTime).fromNow() : ''}</GreyH5>
```

File: packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js (hunk 1 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js]:
```suggestion
```

**Comment by richarddubay** [comment]
@redreceipt I think if you'll check out those changes I suggested earlier and put them in (I think they're relevant) then this would be GTG. 👍

**Comment by redreceipt** [comment]
> @redreceipt I think if you'll check out those changes I suggested earlier and put them in (I think they're relevant) then this would be GTG. 👍

I made those changes :)

**Review by richarddubay — APPROVED** [review]

**Pull request merged**